tests/bud.bats

```diff
@@ -3446,3 +3446,63 @@ _EOF
 
   run_buildah rmi -f ${target}
 }
+
+@test "bud with run should not leave mounts behind cleanup test" {
+  skip_if_in_container
+  run which podman
+  if [[ $status -ne 0 ]]; then
+    skip "podman is not installed"
+  fi
+
+  # Create target dir where we will export tar
+  target=cleanable
+  rm -rf ${target}
+  mkdir ${target}
+
+  # Build and export container to tar
+  run_buildah build --no-cache --signature-policy ${TESTSDIR}/policy.json -t ${target} -f ${TESTSDIR}/bud/containerfile/Containerfile.in ${TESTSDIR}/bud/containerfile
+  podman export $(podman create --name ${target} ${target}) --output=${target}.tar
+
+  # We are done exporting so remove images and containers which are not needed
+  podman rm -f ${target}
+  run_buildah rmi ${target}
+
+  # Explode tar
+  tar -xf ${target}.tar -C ${target}
+  count=$(ls ${target}/run | wc -l)
```

Comment:
This should be `ls -a` or something like it, since one of the things we're trying to make sure isn't being left behind is `/run/.containerenv`.

Reply:
ah sure

Reply:
@nalind using `ls -A` since we don't want `.` and `..`